[2023-12-17 10:00:01,279] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-17T09:00:00+00:00 [queued]>
[2023-12-17 10:00:01,308] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-17T09:00:00+00:00 [queued]>
[2023-12-17 10:00:01,309] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-17 10:00:01,309] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-17 10:00:01,310] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-17 10:00:01,326] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-17T09:00:00+00:00
[2023-12-17 10:00:01,332] {standard_task_runner.py:52} INFO - Started process 60484 to run task
[2023-12-17 10:00:01,337] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-17T09:00:00+00:00', '--job-id', '124', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmpn2_u9huc', '--error-file', '/tmp/tmpyvfmnuqw']
[2023-12-17 10:00:01,341] {standard_task_runner.py:77} INFO - Job 124: Subtask fill
[2023-12-17 10:00:01,413] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-17T09:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-17 10:00:01,866] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-17T09:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-17T09:00:00+00:00
[2023-12-17 10:00:04,558] {python.py:118} INFO - Done. Returned value was: None
[2023-12-17 10:00:04,580] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231217T090000, start_date=20231217T100001, end_date=20231217T100004
[2023-12-17 10:00:04,628] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-17 10:00:04,656] {local_task_job.py:146} INFO - Task exited with return code 0
